packages/flutter_rx_bloc/pubspec.yaml

```diff
@@ -13,15 +13,16 @@ dependencies:
     sdk: flutter
   provider: ^6.0.0
   rx_bloc: ^3.2.0
-  rxdart: ^0.27.2
+  rxdart: ^0.27.3
```

Comment:
Do we use any specific feature from `0.27.3` ?

Reply:
None specific. At the moment of submitting the initial PR, this was the latest version that I updated it to. Other than that, it would be probably safe to even revert if necessary.